src/openzeppelin/tests/token/test_erc721.cairo

```diff
@@ -76,6 +80,7 @@ fn setup() -> ERC721::ContractState {
     let mut state = STATE();
     InternalImpl::initializer(ref state, NAME, SYMBOL);
     InternalImpl::_mint(ref state, OWNER(), TOKEN_ID);
+    testing::pop_log_raw(ZERO());
```

Comment:
i'm starting to think we could abstract this into a `drop_event()` helper or something a bit more explicit to improve tests readability

Reply:
drop_event() looks better indeed.